src/config/configuration_loader.coffee

```diff
@@ -38,11 +37,16 @@ class ConfigurationLoader
 
 
   loadDefaultConfig: (done) =>
+    fsExtra = require 'fs-extra'
     fsExtra.readJson @defaultConfigPath, done
 
 
   loadUserConfig: (done) =>
-    filePaths = _.map extensions, (ext) => path.join @dir, "dependency-lint.#{ext}"
+    async = require 'async'
```

Comment:
Yeah. The only point is if all requires are at the top, its easier to see if another comparable library is already used. Having this buried somewhere on line 73 makes that harder. But this is not a strong point, and it only applies to messy code bases.